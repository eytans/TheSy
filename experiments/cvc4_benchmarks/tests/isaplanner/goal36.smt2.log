05:54:19 [WARN] Collecting statistics
