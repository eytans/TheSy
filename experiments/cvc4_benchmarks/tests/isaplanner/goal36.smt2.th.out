05:54:19 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(take ?x nil) => nil
(take zero ?x2) => nil
(take (succ ?x) (cons ?y ?z)) => (cons ?y (take ?x ?z))
(cons ?y (take ?x ?z)) => (take (succ ?x) (cons ?y ?z))
(takeWhile nil) => nil
nil => (takeWhile nil)
(takeWhile (cons ?x ?y3)) => (ite (P ?x) (cons ?x (takeWhile ?y3)) nil)
(ite (P ?x) (cons ?x (takeWhile ?y3)) nil) => (takeWhile (cons ?x ?y3))
